memphis/consumer.py

```diff
@@ -89,6 +89,11 @@ async def main():
         """
         self.dls_callback_func = callback
         self.t_consume = asyncio.create_task(self.__consume(callback))
+    
+    def stopConsume(self):
```

Comment:
change to stop_consume add add a section on the readme + update the SDK features table on the main memphis repo

Reply:
Please use snake_case for the method names to be consistent.